services/QuillProofreader/src/components/proofreaderActivities/container.tsx

```diff
@@ -14,16 +14,15 @@ import { startListeningToConcepts } from "../../actions/concepts";
 import {
   updateConceptResultsOnFirebase,
   updateSessionOnFirebase,
-  updateSession,
   setSessionReducerToSavedSession
 } from "../../actions/session";
 // import { getConceptResultsForAllQuestions, calculateScoreForLesson } from '../../helpers/conceptResultsGenerator'
 import { SessionState } from '../../reducers/sessionReducer'
 import { ProofreaderActivityState } from '../../reducers/proofreaderActivitiesReducer'
-import { ConceptResultObject } from '../../interfaces/proofreaderActivities'
-import PassageEditor from './passageEditor'
+import { ConceptResultObject, WordObject } from '../../interfaces/proofreaderActivities'
```

Comment:
Not something to deal with right now, but I think that the length of this file (500+ lines even after your deletions here) make it a really good candidate for a refactor.  In my experience, it's pretty hard to keep everything that a file does in your head once it gets near 300 lines, so finding a clean place to split it is usually beneficial.

No need to do that right now on this one, but I figured I'd call it out in case it might be worth a general audit of how many of our files are this size and undertaking a longer-term project to look at them.

Reply:
Yes, that's a good point! I'm not quite sure how to conduct that audit, though, do you have thoughts?

Reply:
There are some tools we could use that would identify our files by size.  Maybe worth using one of those and just starting with the longer files.

Reply:
Sounds like a good plan to me!